remappings.txt

```diff
@@ -0,0 +1,2 @@
+@openzeppelin/=contracts/node_modules/@openzeppelin
```

Comment:
This is necessary for the Solidity VS Code extension to find the source files of the libraries (due to the contracts project being not in the root folder but in a subfolder). Otherwise it complains that it can't find the libraries and it's impossible to click through the library code from our source code.